images/loki/tests/main.tf

```diff
@@ -0,0 +1,58 @@
+terraform {
+  required_providers {
+    oci  = { source = "chainguard-dev/oci" }
+    helm = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_string" "ref" { input = var.digest }
+
+data "oci_exec_test" "test" {
+  for_each = toset(fileset(path.module, "*.sh"))
+
+  digest = var.digest
+  script = "${path.module}/${each.value}"
+}
+
+resource "helm_release" "loki" {
```

Comment:
Is there any good way we can check this once it is up?  Sort of like we use the metrics server here with `kubectl top`: https://github.com/chainguard-images/images/blob/fc964c7de0ebdaa17d242efb517b3052d11b0c78/images/metrics-server/tests/main.tf#L41

Reply:
I've learned that helm charts vary a lot in terms of what "successfully applied" means.  Some return after `kubectl apply` (iirc metrics server) and others do a proper readiness check (iirc prometheus-stack).  If we think this does reasonable checking, then we can maybe forgo additional checks, but we should at least confirm a reasonable readiness check.

Reply:
> I've learned that helm charts vary a lot in terms of what "successfully applied" means. Some return after `kubectl apply` (iirc metrics server) and others do a proper readiness check (iirc prometheus-stack). If we think this does reasonable checking, then we can maybe forgo additional checks, but we should at least confirm a reasonable readiness check.

I checked, this one waits for readiness!